tests/pools/test_pool_rpc.py

```diff
@@ -372,6 +374,66 @@ async def test_create_multiple_pool_wallets(self, one_wallet_node_and_rpc, fee,
         with pytest.raises(ValueError):
             await client.pw_status(3)
 
+        def wallet_is_synced():
+            return (
+                wallet_node_0.wallet_state_manager.blockchain.get_peak_height()
+                == full_node_api.full_node.blockchain.get_peak_height()
+            )
+
+        # Create some CAT wallets to increase wallet IDs
+        for i in range(5):
+            res = await client.create_new_cat_and_wallet(20)
+            await asyncio.sleep(2)
```

Comment:
What are we sleeping here? Just asking because it looks suspicious to become a flaky CI test in case we are waiting for something here.

Reply:
Full node needs to get the TX into the mempool, which we can't check because we don't know tx id here